packages/venia-concept/src/components/Button/button.css

```diff
@@ -1,18 +1,20 @@
 .root {
     composes: root from '../clickable.css';
     background-color: white;
-    border: 1px solid currentColor;
-    border-radius: 1.5rem;
-    color: rgb(var(--venia-teal));
+    border: 1px solid black;
+    border-radius: 6.25rem;
+    color: black;
```

Comment:
Note that these are the styles for the base `Button` component, which means this change updates all the buttons in the entire storefront to this style.

Are we no longer using the `--venia-teal` anywhere?

Reply:
@supernova-at This is a good question, at one hand I support your opinion regarding that not every button will have such a style, but on other hand looks like the whole design of application changes it's major color from --venia-teal to black, so it preferable to retain such button in one place, not changing color from --venia-teal to black in each button that need it